golos.charge/golos.charge.cpp

```diff
@@ -130,7 +131,19 @@ void charge::send_charge_event(name user, const balance& state) {
     eosio::event(_self, "chargestate"_n, std::make_tuple(user, state)).send();
 }
 
-EOSIO_DISPATCH(charge, (use)(useandstore)(removestored)(setrestorer) )
+void charge::useandnotify(name user, symbol_code token_code, uint8_t charge_id, int64_t id, int64_t price_arg, name code, name action_name) {
```

Comment:
I think `user, token_code, charge_id, price_arg, code, action_name, id` is better, because arguments are ordered by purpose.
Let's pass limit argument and send action if new_val more/less limit. So, two actions `usenotifymore`/`usenotifyless` with we implemented instead of `useandnotify`. This way we will save time in many cases.